sys-fs/tmsu/tmsu-0.7.4.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_PN="github.com/oniony/TMSU/"
+DESCRIPTION="Files tagger and virtual tag-based filesystem"
+HOMEPAGE="https://github.com/oniony/TMSU/wiki"
+SRC_URI="https://github.com/oniony/TMSU/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test zsh-completion"
+
+RDEPEND="
+	zsh-completion? ( app-shells/zsh )
+"
+DEPEND="
+	dev-go/go-sqlite3
+	dev-lang/go
+	dev-libs/go-fuse
+"
+
+src_unpack() {
+	default
+	mv TMSU-${PV} ${P} || die "Failed to move sorce directory."
+}
+
+src_install() {
+	dobin misc/bin/*
+	doman misc/man/tmsu.*
+	newbin TMSU tmsu
+
+	if use zsh-completion ; then
```

Comment:
why install a small file conditionally?

Reply:
because unless zsh is enabled we cannot assume that the destinations are available, and we don't want a package which is not zsh to create zsh's directories.

Reply:
we stopped use flagging small files,really. peaple can use the INSTALL_MASK if needed

Reply:
so you suggest just removing the if statement?

Reply:
yes, alsonside wuth the useflag